openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -299,6 +300,49 @@ def create_standardized_media_popularity_function(
     postgres.run(query)
 
 
+def create_audioset_view_query():
+    """
+    Returns SQL to create the audioset_view.
+    """
+    return dedent(
+        f"""
+        CREATE VIEW public.{AUDIOSET_VIEW_NAME}
```

Comment:
We should really have this defined somewhere as a source of truth 😓 Not necessary as part of this PR though, but perhaps we should make an issue for it! Also, do you mind changing the `cast` syntax to match what's being set up in https://github.com/WordPress/openverse-api/pull/648?

Reply:
> We should really have this defined somewhere as a source of truth

Yeah, I was thinking about that. At the moment we're doing the same here for the audio view and image view -- maybe something that can be addressed in #453 while we're refactoring things.

I forgot about the cast syntax! Updated :)